addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -106,11 +106,20 @@ def _prepare_mail_values(self, slide_channel_partner):
         email_layout_xmlid = self.env.context.get('default_email_layout_xmlid', self.env.context.get('notif_layout'))
         if email_layout_xmlid:
             # could be great to use ``_notify_by_email_prepare_rendering_context`` someday
+            message = self.env['mail.message'].sudo().new({'body': mail_values['body_html'], 'record_name': self.channel_id.name})
```

Comment:
This has been improved to use ``_render_encapsulate``, that has also been improved to include more variables used in notification template. But globally it is easy to adapt, just use ``add_context`` for values that are not default :) .